django/core/serializers/base.py

```diff
@@ -20,7 +20,14 @@ class SerializationError(Exception):
 
 class DeserializationError(Exception):
     """Something bad happened during deserialization."""
-    pass
+
+    @classmethod
+    def WithData(cls, original_exc, model, fk, field_value):
```

Comment:
Yes, it is.
from the [Coding Style](https://docs.djangoproject.com/en/1.7/internals/contributing/writing-code/coding-style/) guidelines:
> Use InitialCaps for class names (or for factory functions that return classes).